test/suite_implementation.js

```diff
@@ -69,7 +69,8 @@ module.exports = function(style, options, _callback) {
                 [];
 
             map.remove();
-            gl.destroy();
+            gl.getExtension('STACKGL_destroy_context').destroy();
```

Comment:
What's the difference between the before and after here, and what prompted the change?